tests/algorithm/scripts/send_test_records.py

```diff
@@ -0,0 +1,66 @@
+import csv
+
+import requests
+from helpers import dict_to_pii
+
+
+def send_test_records(test_csv, algorithm_name, api_url):
+    output_data = []
+
+    print("Sending test records to the API...")
```

Comment:
I assume these messages get printed to the CLI? What's the difference between printing vs. logging in this case?

Reply:
The print statements here are meant to give the user an idea of what is happening while they are waiting for the tests to run. It also will print out any error points along the way. Such as a failed batch seeding or records that have errors being linked. All just informational for the user during the testing process. Logging would ideally be more structured and provide more context with each message.